build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     jcenter()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:4.2.2'
+    classpath 'com.android.tools.build:gradle:7.0.0'
```

Comment:
I'm addressing the issues with the Gradle plugin in this PR: https://github.com/medic/cht-android/pull/202

So after merge it you will need to rollback these changes and get the fix from there.

Reply:
modified